templates/repo/home.tmpl

```diff
@@ -5,7 +5,7 @@
 		{{template "base/alert" .}}
 		<div class="ui repo-description">
 			<div id="repo-desc">
-				{{if .Repository.DescriptionHTML}}<span class="description has-emoji">{{.Repository.DescriptionHTML}}</span>{{else if .IsRepositoryAdmin}}<span class="no-description text-italic">{{.i18n.Tr "repo.no_desc"}}</span>{{end}}
+				{{if .Repository.DescriptionHTML}}<span class="description has-emoji">{{.Repository.DescriptionHTML}}</span>{{else}}<span class="no-description text-italic">{{.i18n.Tr "repo.no_desc"}}</span>{{end}}
```

Comment:
@lunny You accidentally removed #2167 here.

Reply:
I addressed this on #5836

Reply:
Why we need ` if .IsRepositoryAdmin`?

Reply:
The idea was to not display the "No Description" message to non-admins. That is, only the repo admin should see it.